rollup.config.js

```diff
@@ -54,5 +54,15 @@ export default [
     },
     external: ["http", "https"],
     plugins: createPlugins()
+  },
+  {
+    input: "lib/entry-umd.ts",
+    output: {
+      format: "iife",
+      name: MODULE_NAME,
+      file: `./dist/${LIBRARY_OUTPUT_NAME}.iife.min.js`
+    },
+    external: ["http", "https"],
```

Comment:
how come this code even needs to be externalised, does the umd build import http and https??

Reply:
When building for node, the http requests are fired by [http/https modules](https://github.com/algolia/search-insights.js/blob/master/lib/utils/request.ts#L29-L30).
I guess this makes sure they're never included in the bundle.